README.md

```diff
@@ -47,8 +47,8 @@ While it may seem strange to see these two in the same file, this is one of the
 
 Let's break down each function:
 
-- `renderToString(source: string, components: object)` - This function consumes a string of mdx along with any components it utilizes in the format `{ ComponentName: ActualComponent }`. The function returns an object that is intended to be passed into `hydrate` directly.
-- `hydrate(source: object, components: object, options?: object)` - This function consumes the output of `renderToString` as well as the same components argument as `renderToString`. It also can optionally be passed options which are [passed directly to mdx](https://mdxjs.com/advanced/plugins). Its result can be rendered directly into your component. This function will initially render static content, and hydrate it when the browser isn't busy with higher priority tasks.
+- `renderToString(source: string, components: object, options?: object)` - This function consumes a string of mdx along with any components it utilizes in the format `{ ComponentName: ActualComponent }`. The function returns an object that is intended to be passed into `hydrate` directly. It also can optionally be passed options which are [passed directly to mdx](https://mdxjs.com/advanced/plugins), `options.remarkPlugins` for reamrk plugins, `options.rehypePlugins` for rehype plugins, and `options.Provider` for custom provider (e.g. [emotion theme provider](https://emotion.sh/docs/theming))
+- `hydrate(source: object, components: object)` - This function consumes the output of `renderToString` as well as the same components argument as `renderToString`. Its result can be rendered directly into your component. This function will initially render static content, and hydrate it when the browser isn't busy with higher priority tasks.
```

Comment:
It seems `options` should place to `renderToString`, could you confirm that? @jescalan

Reply:
That's correct, this was fixed upstream though, so it can be rebased out